internal/pkg/deploy/cloudformation/task.go

```diff
@@ -48,3 +50,62 @@ func (cf CloudFormation) DeployTask(input *deploy.CreateTaskResourcesInput, opts
 
 	return nil
 }
+
+// ListTaskStacks returns all the CF stacks which represent one-off copilot tasks in a given application's environments
```

Comment:
nit: period at the end of the sentence 🙈  (https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)